src/App.js

```diff
@@ -1,13 +1,79 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import './App.css';
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props) {
+    super(props);
+    const savedTasksJson = window.localStorage.getItem('savedTasks');
+    this.state = { 
+      todosList: savedTasksJson ? JSON.parse(savedTasksJson) : [],
+      formField: "",
+    };
+  }
+
+  clickHandler = event => {
+    event.preventDefault();
+    if (event.target.id === 'addTaskButton'){
```

Comment:
This is fine, personally I would have break this in 2 functions though. 
In my own opinion it would have been cleaner.